tools/cythonize.py

```diff
@@ -100,6 +100,24 @@ def process_tempita_pyx(fromfile, tofile):
         f.write(pyxcontent)
     process_pyx(pyxfile, tofile)
 
+
+def process_tempita_pxi(fromfile, tofile):
+    try:
+        try:
+            from Cython import Tempita as tempita
```

Comment:
My point was that I didn't see any actual supporting documentation to say that Cython's vendorization was unreliable besides @rkern's hear-say.